src/lib.rs

```diff
@@ -216,6 +219,9 @@ pub use world::{Component, CreateIter, CreateIterAtomic, EntitiesRes, Entity, En
 #[cfg(feature = "common")]
 pub mod common;
 
+#[cfg(feature = "serialize")]
+pub mod saveload;
```

Comment:
why is it called `saveload` and not `serialize`/`serial`?

Reply:
Because it is also do some tricky data conversion by functions `save` and `load`.
If you insist I'll rename.